portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+/**
+ *Premade list of comments to pass from server
+ */
+  final private ArrayList<String> comments = new ArrayList<>();
```

Comment:
From the [style guide](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers), the order should be `private final`. Same for below